src/mergeRelatorTermFields.js

```diff
@@ -90,7 +80,7 @@ function copyRelatorSubfields(fromField, toField) {
 
 function mergeRelatorTermFields(record, fix = false) {
   /* eslint-disable */
-  // NV: 111/711, 751 and 752 where so rare that I did not add them here
+  // NV: 111/711, 751 and 752 where so rare that I did not add them here. Can't remember why I skipped 6XX and 8XX...
```

Comment:
Format allows use of $e/$j in 6xx/8xx, but our  vocabularies and description conventions do not - so relator terms in these fields are rare / non-existent.